src/main/java/com/ericsson/ei/subscriptionhandler/InformSubscription.java

```diff
@@ -166,8 +180,43 @@ public void informSubscriber(String aggregatedObject, JsonNode subscriptionJson)
     }
 
     /**
-     * This method saves the missed Notification into a single document along with
-     * Subscription name, notification meta and time period.
+     * This method reformats notificationMeta and fetches values if applicable
+     * for the requested parameters.
+     *
+     * @param aggregatedObject
+     * @param notificationMeta
+     * @return String
+     */
+    private String reformatNotificationMeta(String aggregatedObject, String notificationMeta) {
+        String[] splittednotificationMeta = notificationMeta.split("\\?");
+        String URL = splittednotificationMeta[0];
```

Comment:
no need to create two variables here, if you can do next:
String URL = notificationMeta.split("\\?")[0];